mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -270,6 +280,11 @@ void CSaveLocSystem::LevelInitPreEntity()
 
         AddSavelocsFromKV(kvMapSavelocs->FindKey(SAVELOC_KV_KEY_SAVELOCS));
     }
+
+    if (g_pSavelocSystem->LoadStartMarks())
```

Comment:
Two things:

1. You're in the saveloc system, no need to use global accessor

2. This should probably be inside the above if block because if a map doesn't have an entry in the kv, it most certainly won't have start marks saved.

Reply:
Woops how did I miss that.